p11-kit/proxy.c

```diff
@@ -1777,6 +2273,10 @@ C_GetFunctionList (CK_FUNCTION_LIST_PTR_PTR list)
 				rv = CKR_GENERAL_ERROR;
 
 			} else {
+				if (version) {
+					module->version.major = version->major;
+					module->version.minor = version->minor;
```

Comment:
nit: why not just assign like the above line 2265? `module->version = *version;`?